tools/npy_tempita/_looper.py

```diff
@@ -77,53 +77,53 @@ def __repr__(self):
         return '<loop pos=%r at %r>' % (
             self.seq[self.pos], self.pos)
 
+    @property
     def index(self):
         return self.pos
-    index = property(index)
 
+    @property
     def number(self):
         return self.pos + 1
-    number = property(number)
 
+    @property
     def item(self):
         return self.seq[self.pos]
-    item = property(item)
 
+    @property
     def __next__(self):
```

Comment:
This looks very wrong, but was just as wrong before this patch, so fine to leave it.